src/include/connection.h

```diff
@@ -423,6 +423,20 @@ struct __wt_connection_impl {
     uint64_t ckpt_write_bytes;
     uint64_t ckpt_write_pages;
 
+    /* Record the important timestamps of each stage in recovery. */
+    struct __wt_recovery_timeline {
+        uint64_t log_replay;
+        uint64_t rts;
+        uint64_t checkpoint;
+        uint64_t recovery;
+    } recovery_timeline;
+
+    /* Record the important timestamps of each stage in shutdown. */
+    struct __wt_shutdown_timeline {
+        uint64_t rts;
+        uint64_t checkpoint;
+        uint64_t shutdown;
+    } shutdown_timeline;
```

Comment:
Should the names end in `_ms`? It looks easy to use this incorrectly.

Reply:
Yes, changed as suggested.